mws/apis/inbound_shipments.py

```diff
@@ -429,7 +429,7 @@ def put_transport_content(self, shipment_id, is_partnered, shipment_type, **kwar
         else:
             data = self._transport_details_non_partnered(shipment_type, data, **kwargs)
 
-        return self.make_request(data, method="POST")
+        return self.make_request(data, method="POST", use_body_params=True)
```

Comment:
Just to double check - this would make it so that we're only using the alternate behaviour when interacting with the MWS Inbound Shipments API, correct?

Reply:
Yes exactly, and only that specific function from it.